### ISSUE TITLE
Release of the bigger models :)

### ISSUE THREAD

**Issue by christophschuhmann** [jishengpeng/WavTokenizer#1]
Hey, 
I am Christoph one of the co-founders of LAION.

We are working on open source Models like gpt4o and a looking for a better Audio Codec than Snac, which has some problems with very expressive speech and out of distribution sound and music. 

Therefore we are very happy and hopeful about your release! :)

Can you give us some timeline for the release of the bigger models?

If you like we can even collaborate on training even better models on more data, we have lots of data. :)

**Comment by jishengpeng** [comment]
> Hey, I am Christoph one of the co-founders of LAION.
> 
> We are working on open source Models like gpt4o and a looking for a better Audio Codec than Snac, which has some problems with very expressive speech and out of distribution sound and music.
> 
> Therefore we are very happy and hopeful about your release! :)
> 
> Can you give us some timeline for the release of the bigger models?
> 
> If you like we can even collaborate on training even better models on more data, we have lots of data. :)

Thank you very much for your interest!

Prior to our submission on October 2nd, we will release both the medium and large checkpoints. Due to limited computational resources, we are constrained to training each version using only 8 A800 GPUs, which results in a slower training process.

Our training code is open-source, and we invite you to use larger datasets for training. We highly encourage the community to release more powerful checkpoints with more training data.

Should you encounter any issues during the training or inference processes, please feel free to reach out. We are committed to responding promptly and resolving any problems to the best of our ability.

Best regards.

**Comment by Arnold134777** [comment]
> > Hey, I am Christoph one of the co-founders of LAION.
> > We are working on open source Models like gpt4o and a looking for a better Audio Codec than Snac, which has some problems with very expressive speech and out of distribution sound and music.
> > Therefore we are very happy and hopeful about your release! :)
> > Can you give us some timeline for the release of the bigger models?
> > If you like we can even collaborate on training even better models on more data, we have lots of data. :)
> 
> Thank you very much for your interest!
> 
> Prior to our submission on October 2nd, we will release both the medium and large checkpoints. Due to limited computational resources, we are constrained to training each version using only 8 A800 GPUs, which results in a slower training process.
> 
> Our training code is open-source, and we invite you to use larger datasets for training. We highly encourage the community to release more powerful checkpoints with more training data.
> 
> Should you encounter any issues during the training or inference processes, please feel free to reach out. We are committed to responding promptly and resolving any problems to the best of our ability.
> 
> Best regards.

When will large checkpoints be released？

**Comment by systemdevart** [comment]
Does anyone know how to properly scale the model? Is it worth increasing the encoder parameters (since the current one only has 8M) or the decoder parameters (which have around 60M)? And how should other aspects like loss weights or learning rate be adjusted?
